### PR TITLE
post-processor/manifest: add artifact file size to manifest

### PR DESCRIPTION
**Pull request by vtolstov**
Signed-off-by: Vasiliy Tolstov <v.tolstov@selfip.ru>

---
EDIT @rickard-von-essen
Also allow using only the base path of the file.

### CODE REVIEW

**Comment by vtolstov** [comment]
@cbednarski does it looks right?

**Comment by vtolstov** [comment]
after this pr, files section looks like:
```
"files": [
        {
          "name": "output/fedora-23-x86_32-qemu/fedora-23-x86_32",
          "size": 404226048
        },
        {
          "name": "output/fedora-23-x86_32-qemu/fedora-23-x86_32.md5sums",
          "size": 50
        }
      ]
```

**Comment by vtolstov** [comment]
seconf question - i need only file name without path, may be we need some bool variable to do that?

**Comment by vtolstov** [comment]
@cbednarski  @rickard-von-essen can somebody cat review this?

**Comment by rickard-von-essen** [comment]
I'll have a look when I have some time over.

File: website/source/docs/post-processors/manifest.html.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ You can specify manifest more than once and write each build to its own file, or
 ### Optional:
 
 -   `filename` (string) The manifest will be written to this file. This defaults to `packer-manifest.json`.
+-   `filepath_base` (bool) Write only last component from artifact path. This defaults to false.
```
Comment (rickard-von-essen) [inline website/source/docs/post-processors/manifest.html.md]: I would like to rename this and have a simpler explanation, maybe:

> `strip_path` (bool) Write only filename without the path to the manifest file. This defaults to false.

**Comment by rickard-von-essen** [comment]
Adjust the code and docs to the comment above and I'll merge.

**Comment by vtolstov** [comment]
@rickard-von-essen thanks! fixed

File: website/source/docs/post-processors/manifest.html.md [hunk 2]
```diff
@@ -32,7 +33,8 @@ You can simply add `{"type":"manifest"}` to your post-processor section. Below i
   "post-processors": [
     {
       "type": "manifest",
-      "filename": "manifest.json"
+      "filename": "manifest.json",
+      "filepath_base": true
```
Comment (rickard-von-essen) [inline website/source/docs/post-processors/manifest.html.md]: Missed this

**Comment by rickard-von-essen** [comment]
Great! Just one miss in the example code.

**Comment by vtolstov** [comment]
Fixed

**Pull request merged**